<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLobbyDialog</class>
 <widget class="QDialog" name="CreateLobbyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Chat Lobby</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/logo/logo_32.png</normaloff>:/images/logo/logo_32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderFrame" name="headerFrame"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="StyledLabel" name="lobbyInfoLabel">
        <property name="text">
         <string>A chat lobby is a decentralized and anonymous chat group. All participants receive all messages. Once the lobby is created you can invite other friends from the Friends tab.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="lobbyFrame">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Lobby name:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lobbyName_LE"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Lobby topic:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lobbyTopic_LE"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Identity to use:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Visibility:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QComboBox" name="security_CB">
               <item>
                <property name="text">
                 <string>Public (Visible by friends)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Private (Works on invitation only)</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="GxsIdChooser" name="idChooser_CB"/>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="pgp_signed_CB">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you check this, only PGP-signed ids can be used to join and talk in this lobby. This limitation prevents anonymous spamming as it becomes possible for at least some people in the lobby to locate the spammer's node.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>require PGP-signed identities</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Security:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Select the Friends with which you want to group chat.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDockWidget" name="contactsdockWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>52487</width>
              <height>524287</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="floating">
             <bool>false</bool>
            </property>
            <property name="features">
             <set>QDockWidget::NoDockWidgetFeatures</set>
            </property>
            <property name="windowTitle">
             <string>Invited friends</string>
            </property>
            <widget class="QWidget" name="dockWidgetContents">
             <layout class="QGridLayout" name="_2">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="FriendSelectionWidget" name="keyShareList" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>4</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1677215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledLabel.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FriendSelectionWidget</class>
   <extends>QWidget</extends>
   <header>gui/common/FriendSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GxsIdChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsIdChooser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
